erpnext_shipping/erpnext_shipping/doctype/sendcloud/sendcloud.py

```diff
@@ -224,42 +336,14 @@ def get_carrier(self, carrier_name, post_or_get=None):
 		else:
 			return carrier_name.upper() if post_or_get == "get" else carrier_name.lower()
 
-	def get_parcel_dict(
-		self,
-		shipment,
-		parcel,
-		index,
-		delivery_company_name,
-		delivery_address,
-		delivery_contact,
-		service_info,
-		description_of_content,
-		value_of_goods,
-	):
+	def get_parcel(self, parcel, shipment, index, description_of_content, value_of_goods):
```

Comment:
```suggestion
	def get_parcel(self, shipment, parcel, index):
```

- Remove unused parameters
- Keep remaining parameters in the original order

Reply:
You also need to take care of arguments when this method is called. Currently they are in the wrong order.